manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Operation.java

```diff
@@ -0,0 +1,95 @@
+package com.redhat.service.smartevents.manager.v2.persistence.models;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.redhat.service.smartevents.infra.v2.api.models.OperationType;
+
+@Entity
+@Table(name = "OPERATION")
+public class Operation {
+
+    @Id
+    private String id = UUID.randomUUID().toString();
+
+    @Column(name = "type", nullable = false)
+    @Enumerated(EnumType.STRING)
+    private OperationType type;
+
+    @Column(name = "requested_at", columnDefinition = "TIMESTAMP", nullable = false)
+    private ZonedDateTime requestedAt;
+
+    @Column(name = "managed_resource_id", nullable = false)
+    private String managedResourceId;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "operation")
```

Comment:
I'm not sure if `LAZY` is the correct here: if I have understood correctly the "final" `ManagedResourceStatus` is going to be dynamically computed based on the conditions so essentially every time we fetch a resource we will have this:
`Bridge`/`Processor` -> `Operation` ->* `Condition`

So we can consider `EAGER` here (even if it has other drawbacks) 🤔

Reply:
@danielezonca @r00ta and myself discussed this. When we get the list of ALL resources to return to the User we will probably use a different HQL query to retrieve the batch resultset eagerly populated to place the burden of joins on the database.

Reply:
If we're going to use the HQL query to eager fetch the data let's remove the `OneToMany` mapping altogether, it's not mandatory